CONTRIBUTING.md

```diff
@@ -366,8 +372,7 @@ In many cases, you don't really need to build the entire project. Say you want t
 
 ```console
 $ ./install.sh
-$ cd packages/@aws-cdk/aws-ec2
-$ ../../../scripts/buildup
+$ ./build --scope @aws-cdk/aws-ec2 --skip-test --skip-api-check
```

Comment:
`shell
$ cd packages/@aws-cdk/aws-ec2
$ ../../../build.sh --skip-test --skip-api-check
```

Reply:
Okay, I am fine with this too